app/src/main/res/drawable/ic_view_flipper_2.xml

```diff
@@ -0,0 +1,60 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="800dp"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/6c09cd48d00137585ef42b09da4d94f399394e06/app/src/main/res/drawable/ic_view_flipper_2.xml#L2"></span>Limit vector icons sizes to 200×200 to keep icon drawing fast; see https://developer.android.com/studio/write/vector-asset-studio#when for more</td>
    </tr>
  </tbody>
</table>